Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -343,7 +343,7 @@ def check_for_iconv
           nil
         end
         if libiconv&.linked_keg&.directory?
-          unless libiconv.keg_only?
+          unless libiconv&.keg_only?
```

Comment:
```suggestion
          if !...
```

would be a bit easier to follow

Reply:
Pretty sure RuboCop prefers `unless` here.

Reply:
Give it a go but ignore me if Rubocop disagrees.

Reply:
Yeah, RuboCop disagrees.

Reply:
@reitermarkus Cool, thanks for checking, ship as-is!